dm/tests/shardddl_optimistic/conf/dm-master.toml

```diff
@@ -0,0 +1,10 @@
+# Master Configuration.
+name = "master1"
+master-addr = ":8261"
+advertise-addr = "127.0.0.1:8261"
+peer-urls = "127.0.0.1:8291"
+initial-cluster = "master1=http://127.0.0.1:8291"
+auto-compaction-retention = "3s"
+
+[experimental]
```

Comment:
now we can remove the `experimental` label

Reply:
keep as compatibility test